components/import-action/menu/menu.module.scss

```diff
@@ -0,0 +1,70 @@
+.importDropdown {
+  z-index: 1;
```

Comment:
is this necessary?

Reply:
removed, but I increased the menu z-index so that it goes over the dependency graph